ui/src/testing/stubs/cdr-versions-api-stub.ts

```diff
@@ -1,53 +1,72 @@
 import {AccessTierShortNames} from 'app/utils/access-tiers';
-import {ArchivalStatus, CdrVersion, CdrVersionListResponse, CdrVersionsApi} from 'generated/fetch';
+import {
+  ArchivalStatus,
+  CdrVersionsApi,
+  CdrVersionTiersResponse
+} from 'generated/fetch';
 import {stubNotImplementedError} from 'testing/stubs/stub-utils';
 
 export class CdrVersionsStubVariables {
   static DEFAULT_WORKSPACE_CDR_VERSION = 'Fake CDR Version';
   static DEFAULT_WORKSPACE_CDR_VERSION_ID = 'fakeCdrVersion';
   static ALT_WORKSPACE_CDR_VERSION = 'Alternative CDR Version';
   static ALT_WORKSPACE_CDR_VERSION_ID = 'altCdrVersion';
+  static CONTROLLED_TIER_CDR_VERSION = 'Controlled Tier CDR Version';
+  static CONTROLLED_TIER_CDR_VERSION_ID = 'ctCdrVersion';
 }
 
-export const cdrVersionListResponse: CdrVersionListResponse = {
-  defaultCdrVersionId: CdrVersionsStubVariables.DEFAULT_WORKSPACE_CDR_VERSION_ID,
-  items: [
+export const cdrVersionTiersResponse: CdrVersionTiersResponse = {
```

Comment:
opt: existing issue, but I would generally define test data like this with a function instead of a static variable. With the current approach, all it takes is one test method to modify this data, and then it's tainted for all subsequent runs, which can be tricky to track down. (Note: I may have been the one to originally define this as a static variable)

One idea: perhaps this is a good use case for the `get` keyword, which conveniently contains the change to just this file as well.

Reply:
Note: seems you actually can't define a getter at the top level without doing some shenanigans, so ignore my idea.